modules/eco-configuring-control-plane-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
```

Comment:
Apologies for the confusion here. Initially I had a new module called
eco-configuring-control-plane-machine-health-check-with-self-node-remediation.adoc
but on reinspection, and because the content was so similar, we combined the content into the existing module called 
eco-configuring-machine-health-check-with-self-node-remediation.adoc